README.md

```diff
@@ -165,6 +168,51 @@ yotiClient.getActivityDetails(token).then((activityDetails) => {
 
 [Example reference](examples/profile/index.js)
 
+
+## Programmatic QR code creation
+
+Dynamic Policy generation is a way of requesting non-static attribute lists for your application.
+
+This is useful when you want or need different permutations of attributes from Yoti, without having to create a new application.
```

Comment:
Should this be `without having to create a new scenario`? I guess you could also create a new application, but seems a bit overkill.
```suggestion
This is useful when you want or need different permutations of attributes from Yoti, without having to create a new scenario.
```